aws/dms/tasks.yml

```diff
@@ -1,3 +1,25 @@
+# These configuration settings are utilized by data.yml.erb to generate configuration of the AWS Database Migration
+# Service (DMS) Tasks. Merging changes to this file does NOT update the `DATA-production` Stack that provisions
+# our AWS Redshift Cluster and the DMS Replication Tasks that export data into the analytics system. Before merging
+# a Pull Request that changes this file, an Infrastructure Engineer must execute the following steps on their workstation
+# utilizing their admin AWS credentials to validate and then deploy these changes:
```

Comment:
Is it worth adding an example of what a validation error looks like, or specifying exactly what in the output the developer should be looking for to determine that validation was successful? I expect it should be relatively obvious, but might be nice to be explicit anyway.

Reply:
Definitely worth doing that. I added a small explanation of what expected an unxpected output of the validation Task might be.